apps/web/vibes/2px/components/radio-button/index.tsx

```diff
@@ -0,0 +1,44 @@
+import clsx from 'clsx'
+
+interface RadioButtonProps extends Omit<React.HTMLProps<HTMLInputElement>, 'type' | 'hidden'> {
```

Comment:
@LHDi One thing I just recalled: in the [development guide](https://linear.app/tinloof/document/development-guide-bd8c365e62de), it says the components should not receive react element props. Do you think we can keep only the data props here?

Reply:
it's not passing react elements here @freadbaker

Reply:
I had gotten this restriction wrong. I'm resolving this conversation, then.